js/recipe/image.js

```diff
@@ -0,0 +1,16 @@
+import { htmlElement } from "../htmlRenderer.js"
+import { render } from "../htmlRenderer.js"
+
+export function image(recipe) {
+
+  const imageAttributes = {
+    src: recipe.image,
+    alt: recipe.name,
+    width: "900px",
+    style: "padding-top: 20px;"
+  }
+
+  const recipeParentElement = document.querySelector("article.post.single");
+  const imageDiv = render(recipeParentElement, htmlElement("div")({className: "recipe-image"}));
+  render(imageDiv, htmlElement("img")({ attributes: imageAttributes }));
```

Comment:
I believe that this can be passed in as:

```js
htmlElement("div")({className: "recipe-image", children: htmlElement("img")({ attributes: imageAttributes }) });
```

Reply:
This is an example of a "component-tree", where each component may or may not contain other components.

Reply:
Yep. I did the same thing when I was refactoring metadata.js, only calling `render` once and then passing all the child elements in at the same time. There is no need for this now however, since I am rendering the image component from recipe.js